### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by radoslawo81** [grillbaer/spectracle#14] [Q&A]
Hello
Could you put short instalation instruction for beginners? The aplication look very interesting but i dont know how to run this in Windows or Linux. Thank You in adwance.

**Comment by grillbaer** [comment]
Hi,

I just triggered a fresh build on github because the last installer files seem to have timed out and were automatically deleted by github.
=> You'll find a Windows MSI installer for download under [Actions ](https://github.com/grillbaer/spectracle/actions), at the very bottom of the page after you select the latest (and currently only) [workflow run](https://github.com/grillbaer/spectracle/actions/runs/10441732633):

Direct download link: [Binary-Windows.zip](https://github.com/grillbaer/spectracle/actions/runs/10441732633/artifacts/1824792892)

The zip file contains an MSI installer. Extract it from the zip archive and execute the .msi file. 
After installation there should be a Spectracle icon on your desktop.

Best regards,
Holger

**Comment by radoslawo81** [comment]
Very very thank You for fast reaction and help.
